lighthouse-core/lib/i18n.js

```diff
@@ -20,6 +19,8 @@ try {
 
   // @ts-ignore
   const IntlPolyfill = require('intl');
+  if (!IntlPolyfill.NumberFormat) throw new Error('Invalid polyfill');
```

Comment:
@paulirish FYI this was my fix to what you discovered in #5702

Reply:
ah cool. nice. 

The msg is a little weird though. Seems like this case is described as: "Polyfill is a no-op due to existing implementation". The fact that we exit via throwing is a little random. can we do a IIFE and return instead?

Reply:
yeah IFFE makes a lot more sense, done 👍